.happy/terraform/modules/sfn_config/gisaid.wdl

```diff
@@ -177,13 +177,12 @@ task TransformGISAID {
     git clone --depth 1 https://github.com/nextstrain/ncov-ingest /ncov-ingest
     ncov_ingest_git_rev=$(git -C /ncov-ingest rev-parse HEAD)
 
-    # modify location rules from ncov-ingest. Southern San Joaquin Valley would be left blank in the default version
-    sed -i -e 's/Southern San Joaquin Valley\tNorth America\/USA\/California\//Southern San Joaquin Valley\tNorth America\/USA\/California\/Tulare County/' \
-    -e 's/Orange County CA/Orange County/' \
-    -e 's/Monterey County CA/Monterey County/' \
-    -e 's/Alameda County CA/Alameda County/' \
-    -e '/Illinois\/Chicago\tNorth America/d' \
-    /ncov-ingest/source-data/gisaid_geoLocationRules.tsv
+    # modify location rules from ncov-ingest based on the locations we're using. 
+    python3 /usr/src/app/aspen/workflows/transform_gisaid/update_locations.py \
+            --input /ncov-ingest/source-data/gisaid_geoLocationRules.tsv      \
+            --output /ncov-ingest/source-data/czgenepi_locations.tsv
+    mv /ncov-ingest/source-data/czgenepi_locations.tsv \
+       /ncov-ingest/source-data/gisaid_geoLocationRules.tsv
```

Comment:
Here's where we mangle the geoLocationRules file with our new script instead of `sed`